{"name": "test_click_cart[Chrome]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <Test.test_homepage.Test_HomePage object at 0x0000022D33B76E00>\n\n    def test_click_cart(self):\n        self.home = Homepage(self.driver)\n>       self.home.do_get_cart_count()\n\nTest\\test_homepage.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nPages\\Homepage.py:79: in do_get_cart_count\n    self.do_click(self.cartCount)\nPages\\BasePages.py:18: in do_click\n    WebDriverWait(self.driver,30).until(EC.visibility_of_element_located(by_locator)).click()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"01ffc5e93b67a44c5dae278483f75197\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x0000022D33C0ACB0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nvenv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException"}, "parameters": [{"name": "init_driver", "value": "'Chrome'"}], "start": 1690959741251, "stop": 1690959773257, "uuid": "f5aa763f-bb24-42eb-a34b-8a1782996742", "historyId": "d94c1ff4674fb2fd1f4aedbdf4750c2e", "testCaseId": "0408119091ef7537f070cf01a7df1e10", "fullName": "Test.test_homepage.Test_HomePage#test_click_cart", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "Test"}, {"name": "suite", "value": "test_homepage"}, {"name": "subSuite", "value": "Test_HomePage"}, {"name": "host", "value": "INBAWN163332"}, {"name": "thread", "value": "16440-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test.test_homepage"}]}